homework/Vasili_Zasinets/Shape/src/main/java/com/tutrit/java/quickstart/shape/fabric/Fabric.java

```diff
@@ -0,0 +1,16 @@
+package com.tutrit.java.quickstart.shape.fabric;
+
+public class Fabric {
+
+    //created a private constructor on the recommendation of sonar
```

Comment:
Class Fabric have one static method.
Don't necessary create object of class Fabric.
Changing access modifier on privat forbid create object of Fabric.